influxql/_v1tests/server_helpers.go

```diff
@@ -146,37 +154,32 @@ func (qt *Test) addQueries(q ...*Query) {
 func (qt *Test) init(ctx context.Context, t *testing.T, p *tests.DefaultPipeline) (fx pipeline.BaseFixture, auth *influxdb.Authorization) {
 	t.Helper()
 
-	require.Greater(t, len(qt.writes), 0)
-
 	qt.orgID = p.DefaultOrgID
 	qt.bucketID = p.DefaultBucketID
 
 	fx = pipeline.NewBaseFixture(t, p.Pipeline, qt.orgID, qt.bucketID)
 
-	qt.writeTestData(ctx, t, fx.Admin)
-	p.Flush()
-
-	writeOrg, err := influxdb.NewPermissionAtID(qt.orgID, influxdb.WriteAction, influxdb.OrgsResourceType, qt.orgID)
-	require.NoError(t, err)
-
-	bucketWritePerm, err := influxdb.NewPermissionAtID(qt.bucketID, influxdb.WriteAction, influxdb.BucketsResourceType, qt.orgID)
-	require.NoError(t, err)
-
-	bucketReadPerm, err := influxdb.NewPermissionAtID(qt.bucketID, influxdb.ReadAction, influxdb.BucketsResourceType, qt.orgID)
-	require.NoError(t, err)
+	if !qt.noWrites {
+		require.GreaterOrEqual(t, len(qt.writes), 0)
```

Comment:
This was `require.Greater` before, is the change to `GreaterOrEqual` intentional?

Reply:
No, I initially wasn't going to have a `noWrites` flag, and was testing `GreaterOrEqual`, but realised that defeated the purpose, as writes will always be ≥ 0 🤦